packages/base/src/MediaRange.ts

```diff
@@ -83,7 +83,7 @@ const getCurrentRange = (name: string, width = window.innerWidth): string => {
 	let currentRangeName;
 
 	rangeSet.forEach((value, key) => {
-		if (width >= value[0] && width <= value[1]) {
+		if (Math.floor(width) >= value[0] && Math.floor(width) <= value[1]) {
```

Comment:
Better introduce a new const for "Math.floor(width)" before the loop, because now we call "Math.floor" twice per iteration , while it can be calculated just once.
f.e.
const effectiveWidth = Math.floor(width)

Reply:
Hi Ilhan,

After some *(manual)* testing and debugging, I came to the following conclusion:

Following the logic that `width: 1023px` => "S" size and `width: 1023.5px` => M size, I found out that even without our change with the `Math.floor()`, the `width` value, our method `getCurrentRange` recieves is **always** rounded ( with `Math.round` logic, which returns the bigger integer after .5 ). With that said I even think that we might need to revert the Math.floor() change in order to keep the old logic, since our method never recieves a floating number.

However, if I am mistaken I also think that using the `Math.round()` would be better solution, since for example 
1023.6 is closer to 1024, than to 1023. The only exception here is 1023.5 because it is equal range from either 1023 and 1024. What do you think ?

Please correct me, if Im wrong.

Best regards,
Stoyan